README.md

```diff
@@ -129,10 +129,35 @@ $ node src/app.js
 With the server running, open your favorite web browser and visit:
 http://localhost:3000
 
-If you want to specify the port number, set the `PORT` environment variable like so:
+### Environment Variables
+
+In order to prevent uploading private keys, this repository uses a .env file for storing environment variables.
+
+For example, if you want to specify the port number, set the `PORT` environment variable like so:
+
+```bash
+PORT=8080
+```
+
+To manage this, we will install dotenv in the root directory using npm, and save it as a dependency in our package.json file:
 
 ```bash
-$ PORT=8080 node src/app.js
+$ npm install dotenv --save
 ```
 
+We will also require it at the top of our src/app.js file:
+
+```javascript
+require('dotenv').config()
+```
```

Comment:
These instructions for installing the `dotenv` module and requiring it in **app.js** don't belong in the docs. You've already installed the module, and the `--save` flag added it to the dependencies in **package.json**. Anyone pulling this repository down need only run `npm install` (which is already documented above). Also, you have already required `dotenv` in app.js, so no need to mention that here.

You should, on the other hand, tell the reader how they can set environment variables by creating a `.env` file in the root directory, and assuring them that this file will be git ignored.

I know it's a pain, but explaining the code by documenting it will help you understand how it works as much as it will help future developers who work in your codebase.